cypress/integration/test_table.js

```diff
@@ -155,4 +155,27 @@ describe("Tests Table", function() {
       });
     });
   });
+
+  describe("Test Name Filter", () => {
+    beforeEach(() => {
+      cy.visit(TESTS_ROUTE);
+      cy.get("#cy-testname-input").type("group");
+    });
+    it("Typing in test name filter updates testname query param", () => {
+      cy.location().should(loc => {
+        expect(loc.search).to.include("testname=group");
+      });
+    });
+
+    it("Input value is incuded in the taskTests GQL request body under variables.testName ", () => {
+      waitForTestsQuery(); // this query is for the initial data load before any typing begins
+      waitForTestsQuery();
```

Comment:
it has to be done twice in a row?

Reply:
the first query is the initial load, the second one is after typing

Reply:
I can alter the util function so we can pass in gql variables we're looking for such as testName in this case. also for some reason this test passes locally but not in CI